src/Server/RestRouterHandlers/SearchHandler.cpp

```diff
@@ -75,7 +75,6 @@ String SearchHandler::getQuery(const Poco::JSON::Object::Ptr & payload) const
     const auto & mode = payload->has("mode") ? payload->get("mode").toString() : "verbose";
 
     /// FIXME: to support 'realtime' mode in future
-    /// FIXME: change the behavior of 'verbose' mode. Currently in frontier service, 'standard' and 'verbose' mode are the same
     if (mode == "standard" || mode.empty() || mode == "verbose")
         query_context->setDefaultFormat("JSONCompactEachRowWithNamesAndTypes");
```

Comment:
Response Format is not bound to `mode`, let's move this line to `setQuerySettings`

```
String default_format;
for (const auto & [key, value] : *query_parameters)
{
   ...
}

if (default_format.empty())
{
    default_format = "JSONCompact";
}
query_context->setDefaultFormat(default_format);
```